charts/sentry-apps/values.yaml

```diff
@@ -98,6 +98,10 @@ system:
   # Changing it invalidates all the current sessions.
   secretKey:
 
+# hooks -- configure hooks
+# @default -- See [values.yaml](https://github.com/sentry-kubernetes/charts/tree/develop/sentry) of upstream project
```

Comment:
Basically the same detail then the one in the README.md: If there is no good reason against it, I would link directly to the values.yaml file in the upstream project.
(And most likely also adapt line 93 accordingly.)
```suggestion
# @default -- See [values.yaml](https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml) of upstream project
```